doc/util.md

```diff
@@ -73,15 +73,18 @@ class MyGame extends Game {
   @override
   void update(double dt) {
     countdown.update(dt);
-    if (countdown.isFinished()) {
-      // do something ...
+    if (countdown.finished) {
+      // Prefer the timer callback, but this is better in some cases
```

Comment:
Suggestion: `Usually the timer callback could be used, but this could be used in some cases`